packages/suite-web/scripts/s3sync.sh

```diff
@@ -1,5 +1,5 @@
 #! /usr/bin/env nix-shell
-#! nix-shell -i bash -p awscli
+#! nix-shell -i bash -p awscli2
 
 # Before first use:
 # Install awscli (pip install awscli)
```

Comment:
Maybe it's worth to update the instruction for updating on awscli2? Is it needed to uninstall v1 as they are sharing the same `aws` command name? Or is it working out of the box in nix-shell?

Reply:
this will launch a shell where only awscli2 is present so no need to do anything. Also in case we decide to change this script our deploy runner has this tool preinstalled in version 2.x.x and it should work too.